pkgs/tools/networking/cadaver/default.nix

```diff
@@ -14,11 +14,15 @@ stdenv.mkDerivation rec {
       name = "disable-sslv2.patch";
       sha256 = "1qx65hv584wdarks51yhd3y38g54affkphm5wz27xiz4nhmbssrr";
     })
+    # Cadaver also works with newer versions of neon than stated
+    # in the configure script
+    ./configure.patch
```

Comment:
Is there a way to let upstream know about this?

Reply:
Upstream seems to be rather dead last release was in 2009. Thanks for merging.